xvec/accessor.py

```diff
@@ -918,6 +918,375 @@ def to_geodataframe(
         )
         return df
 
+    def agg_geom(
```

Comment:
```suggestion
    def _agg_geom(
```
This should be private, the public API is `zonal_stats`.